Assets/Scripts/Gameplay/GameState/BossRoomState.cs

```diff
@@ -16,14 +16,11 @@ namespace Unity.Multiplayer.Samples.BossRoom.Server
     /// Server specialization of core BossRoom game logic.
     /// </summary>
     [RequireComponent(typeof(NetcodeHooks))]
-    public class ServerBossRoomState : GameStateBehaviour
+    public class BossRoomState : GameStateBehaviour
```

Comment:
mmm having the prefix did help adding context on where this class was expected to execute. I can be convinced otherwise though

Reply:
Prefix got removed because there's no matching client state anymore, however I'm ambivalent and we can revert this name change.

Reply:
@fernando-cortez @LPLafontaineB what do you guys think?

Reply:
Personally, I'm still keeping the prefix on some classes I'm still creating nowadays. It's nice to know at a glance where this logic is expected to execute.

Reply:
Reverted the names back!